snmp/datadog_checks/snmp/data/default_profiles/chrysalis-luna-hsm.yaml

```diff
@@ -0,0 +1,54 @@
+extends:
+  - chrysalis.yaml
+  - _generic-tcp.yaml
+  - _generic-ucd.yaml
```

Comment:
are there other product specific profiles ?
if not should this be moved to the `chrysalis.yaml` profile ?

Reply:
@NouhaManai96 I asked myself that question too, but I'm not sure if other chrysalis will all support them, hence I prefer to keep them in chrysalis-luna-hsm.yaml for now, we can move then to `chrysalis.yaml` if needed.